README.md

```diff
@@ -69,8 +71,15 @@ pytest tests/ \
     --lg-log \
     --log-cli-level=CONSOLE \
     --lg-colored-steps \
-    --firmware ../../openwrt/bin/targets/malta/be/openwrt-malta-be-vmlinux-initramfs.elf
+    --firmware ../../openwrt/bin/targets/malta/be/openwrt-malta-be-vmlinux-initramfs.elf \
+    --reportportal \
+    -rp-launch="name_of_test" \
```

Comment:
Please remove the last backslash, it would attach an empty line to command. Please read this to get an idea https://unix.stackexchange.com/questions/368753/what-does-this-command-with-a-backslash-at-the-end-do